### PR TITLE
add konrad-malawski-ja-1

### LINKED ISSUE #345: 和訳： Konrad Malawskiさんによる「Distribute All The Things (with Akka)」
ref #339

### PR DESCRIPTION
**Pull request by richardimaoka**
Closes #345 

---
name: Konrad `ktoso` Malawski
title: "Distribute All The Things (with Akka)"
length: 40
audience: Intermediate
language: English
twitter: ktosopl
github: ktoso
icon: https://scontent-nrt1-1.xx.fbcdn.net/v/t31.0-8/15626425_10202875601369012_2764589549729738273_o.jpg?oh=57591212f91d0b834b45ff9806e05ec8&oe=5A921F16
organization: Akka Team, Lightbend
tags:
  - Software Design and Architecture, Microservices, Others

suggestions:
  - Developers interested in building performant and reliable systems. No prior exposure to Akka is needed to enjoy the session.
---
With Akka's recent focus, and accomplishment of shipping Akka Streams and having included Reactive Streams (java.util.concurrent.Flow) in the OpenJDK (Java 9), one has to wonder -- what's the next focus point for the Akka toolkit? It actually means continuing where we left off, and adding more types to all APIs (as was already started by Akka Streams), via the introduction of a stable Akka Typed API as well as more integration between Streams and Cluster. In other words -- allowing to build safe, back-pressure aware distributed systems that type-check and can scale out  (or in) according to your needs. In this session, we'll look at the evolution of Akka's APIs and core principles, and how the new APIs allow us to integrate and distribute all things, for building reactive and distributed systems.

### CODE REVIEW

File: _candidates_ja/KonradMalawski_1.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+twitter: ktosopl
+github: ktoso
+icon: https://scontent-nrt1-1.xx.fbcdn.net/v/t31.0-8/15626425_10202875601369012_2764589549729738273_o.jpg?oh=57591212f91d0b834b45ff9806e05ec8&oe=5A921F16
+organization: Akka Team, Lightbend
+tags:
+  - Software Design and Architecture
+  - Microservices
+  - Others
+suggestions:
+  - 高性能・高信頼性のシステムに興味のある開発者、このセッションを聴講するのにAkkaに関する事前知識は必要はありません
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: Developers interested in building performant and reliable systems. No prior exposure to Akka is needed to enjoy the session.

File: _candidates_ja/KonradMalawski_1.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+name: Konrad `ktoso` Malawski
+title: "あらゆるものを分散コンピューティングで扱おう(Akkaをつかって)"
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: "Distribute All The Things (with Akka)"

File: _candidates_ja/KonradMalawski_1.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+tags:
+  - Software Design and Architecture
+  - Microservices
+  - Others
+suggestions:
+  - 高性能・高信頼性のシステムに興味のある開発者、このセッションを聴講するのにAkkaに関する事前知識は必要はありません
+
+---
+Akkaの最近の動向や、Akka StreamsがリリースされReactive Streams(java.util.concurrent.Flow)がOpenJDK (Java 9)に含まれるという事実によって、
+あなたはもしかするとAkkaの次の開発の焦点は何なのか？と考えているかもしれません。
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: With Akka's recent focus, and accomplishment of shipping Akka Streams and having included Reactive Streams (java.util.concurrent.Flow) in the OpenJDK (Java 9), one has to wonder -- what's the next focus point for the Akka toolkit?

File: _candidates_ja/KonradMalawski_1.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+実はAkkaが次の開発の焦点は、過去に行ったことのなかでまだ完成されていないものです。
+すなわち、全てのAPIの中からより多くのものに型情報を追加することであり(Akka Streamsで実現されたように)、
+そしてこれは安定したAkka Typed APIおよびそのAPIをStreamsとClusterに統合することによって可能になります。
+別の言い方をすれば、ビルド時安全で、型検査を行うことが出来、back-pressureを扱うことができ、あなたのニーズによってスケールアウト・インすることができる
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: It actually means continuing where we left off, and adding more types to all APIs (as was already started by Akka Streams), via the introduction of a stable Akka Typed API as well as more integration between Streams and Cluster.

File: _candidates_ja/KonradMalawski_1.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+分散コンピューティングシステムを実現することです。
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: In other words -- allowing to build safe, back-pressure aware distributed systems that type-check and can scale out (or in) according to your needs.

File: _candidates_ja/KonradMalawski_1.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+このセッションではAkkaのAPIの原則とAPIがどのように変遷してきたか、そして新しいAPIがどのようにあらゆるものを統合し、
+リアクティブな分散コンピューティング環境に置くことが出来るかお話します。
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: In this session, we'll look at the evolution of Akka's APIs and core principles, and how the new APIs allow us to integrate and distribute all things, for building reactive and distributed systems.

File: _candidates_ja/KonradMalawski_1.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  - Others
+suggestions:
+  - 高性能・高信頼性のシステムに興味のある開発者、このセッションを聴講するのにAkkaに関する事前知識は必要はありません
+
+---
+Akkaの最近の動向や、Akka StreamsがリリースされReactive Streams(java.util.concurrent.Flow)がOpenJDK (Java 9)に含まれるという事実によって、
+あなたはもしかするとAkkaの次の開発の焦点は何なのか？と考えているかもしれません。
+実はAkkaが次の開発の焦点は、過去に行ったことのなかでまだ完成されていないものです。
+すなわち、全てのAPIの中からより多くのものに型情報を追加することであり(Akka Streamsで実現されたように)、
+そしてこれは安定したAkka Typed APIおよびそのAPIをStreamsとClusterに統合することによって可能になります。
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: It actually means continuing where we left off, and adding more types to all APIs (as was already started by Akka Streams), via the introduction of a stable Akka Typed API as well as more integration between Streams and Cluster.

File: _candidates_ja/KonradMalawski_1.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+  - 高性能・高信頼性のシステムに興味のある開発者、このセッションを聴講するのにAkkaに関する事前知識は必要はありません
+
+---
+Akkaの最近の動向や、Akka StreamsがリリースされReactive Streams(java.util.concurrent.Flow)がOpenJDK (Java 9)に含まれるという事実によって、
+あなたはもしかするとAkkaの次の開発の焦点は何なのか？と考えているかもしれません。
+実はAkkaの次の開発の焦点は、過去に行ったことのなかでまだ完成されていないものです。
+すなわち、全てのAPIの中からより多くのものに型情報を追加することであり(Akka Streamsで実現されたように)、
+そしてこれは安定したAkka Typed APIおよびそのAPIをStreamsとClusterに統合することによって可能になります。
+別の言い方をすれば、ビルド時安全で、型検査を行うことが出来、back-pressureを扱うことができ、あなたのニーズによってスケールアウト・インすることができる
+分散コンピューティングシステムを実現することです。
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: In other words -- allowing to build safe, back-pressure aware distributed systems that type-check and can scale out (or in) according to your needs.

File: _candidates_ja/KonradMalawski_1.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+  - Others
+suggestions:
+  - 高性能・高信頼性のシステムに興味のある開発者、このセッションを聴講するのにAkkaに関する事前知識は必要はありません
+
+---
+Akkaの最近の動向や、Akka StreamsがリリースされReactive Streams(java.util.concurrent.Flow)がOpenJDK (Java 9)に含まれるという事実によって、
+あなたはもしかするとAkkaの次の開発の焦点は何なのか？と考えているかもしれません。
+実はAkkaの次の開発の焦点は、過去に行ったことのなかでまだ完成されていないものです。
+すなわち、全てのAPIの中からより多くのものに型情報を追加することであり(Akka Streamsで実現されたように)、
+そしてこれは安定したAkka Typed APIおよびそのAPIをStreamsとClusterに統合することによって可能になります。
```
Comment (richardimaoka) [inline _candidates_ja/KonradMalawski_1.md]: It actually means continuing where we left off, and adding more types to all APIs (as was already started by Akka Streams), via the introduction of a stable Akka Typed API as well as more integration between Streams and Cluster.

File: _candidates_ja/KonradMalawski_1.md (hunk 2 above)
Comment (eed3si9n) [reply _candidates_ja/KonradMalawski_1.md]: 訳案: 分散処理に使っていいよ (Akka のすすめ)

Comment (richardimaoka) [reply _candidates_ja/KonradMalawski_1.md]: ありがとうございます！案を頂いたとおりタイトルを変えました。

**Pull request merged**